src/data/faq_de.json

```diff
@@ -1934,6 +1934,29 @@
                             }
                         ]
                     },
+                    {
+                        "title": "Backup",
+                        "id": "support_backup",
+                        "indexed": true,
+                        "accordion":[
+                            {
+                                "duplicate":"eu_dcc_export"
+                            },
+                            {
+                                "duplicate":"vac_cert_multiple_scan_possible"
+                            },
+                            {
+                                "duplicate":"new_device"
+                            },
+                            {
+                                "duplicate":"migration_Apple"
+                            },

```

Comment:
Same comment as before:

- I suggest to remove this reference. It is not longer relevant because CWA iOS 1.5.0 is no longer supported. See issue https://github.com/corona-warn-app/cwa-website/issues/3238